frontend/src/app/testQueue/AoEForm/AoEForm.tsx

```diff
@@ -204,7 +196,9 @@ const AoEForm: React.FC<Props> = ({
   // Auto-answer pregnancy question for males
   const pregnancyResponses = getPregnancyResponses();
   if (patient.gender === "male" && !pregnancyResponse) {
-    setPregnancyResponse(findValueForLabel("No", pregnancyResponses));
+    setPregnancyResponse(
+      findValueForLabel("No", pregnancyResponses) as PregnancyCode
+    );
```

Comment:
I would add a new constant to `constants.ts` `const pregnancyNoCode = "60001007"` and then use `pregnancyNoCode` as the key in `pregnancyMap` and here
```suggestion
    setPregnancyResponse(pregnancyNoCode);
```